.github/workflows/myco_ci.yaml

```diff
@@ -48,4 +62,5 @@ jobs:
           DOCKER_USERNAME: ${{ secrets.DOCKER_USERNAME }}
           DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
           INTEGRATION_TESTS_REPO: https://gsydev:${{ secrets.GSYDEV_TOKEN }}@github.com/gridsingularity/gsy-backend-integration-tests.git
-          INTEGRATION_TESTS_BRANCH: ${{ steps.validated_branch.outputs.INTEGRATION_TESTS_BRANCH }}
+          INTEGRATION_TESTS_BRANCH: ${{ steps.validated_int_tests_branch.outputs.INTEGRATION_TESTS_BRANCH }}
+          TARGET_BRANCH: ${{ steps.validated_target_branch.outputs.TARGET_BRANCH }}
```

Comment:
I'm a bit confused. Where is this variable used? I see we pass it as env variable in the tox env, but why? Is it supposed to interact with the GBIT code?

Reply:
@fievelk 
This variable is the one that is parsed from the PR message as `**INTEGRATION_TESTS_BRANCH**=master `.
It is passed to the tox env to clone the GBIT branch specified on the PR message.

The analogous is made now for TARGET_BRANCH, which would be passed to the tox env and used inside the script which builds the docker image.